deconz/config.yaml

```diff
@@ -25,6 +25,10 @@ init: false
 kernel_modules: true
 options:
   device: null
+  ota_update:
+    bosch: false
+    ikea: false
+    ledvance: false
```

Comment:
From what I understand, these were unconditionally enabled so far, and with that they are disabled by default now. Is there a particular reason for this?

I am fine defaulting to disabled, but I'd mention that explicitly in the changelog.